OxfordBodleian/Juel-Jensen/BDLaethd19.xml

```diff
@@ -456,26 +506,958 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d8" type="miniature">
+                                <locus target="#147r"/>
+                                <desc></desc>
+                                <q xml:lang="gez"><unclear>ማ</unclear>ትያስ፡</q>
+                                <q xml:lang="gez">ታዴዎስ<unclear>፡</unclear></q>
+                                <q xml:lang="gez">ያዕቆብ፡ ወልደ፡ እልፍዮስ።</q>
+                                <q xml:lang="gez">ያዕቆብ፡ ወልደ፡ ዘብዴዎስ፡</q>
+                                <q xml:lang="gez"><unclear>ና</unclear>ት<unclear></unclear><gap reason="lost" unit="chars" quantity="1">
+                                </gap>ኤል<supplied reason="omitted">፡</supplied></q>
+                                <q xml:lang="gez">ሉቃስ፡</q>
+                                <q xml:lang="gez">ፊልሞና፡</q>
+                                <q xml:lang="gez">ቀሌምንጦስ</q>
+                                <q xml:lang="gez">ዮሴፍ</q>
+                                <q xml:lang="gez">ኒቆዲሞስ።</q>
+                                <q xml:lang="gez">አብሮኮስ፡</q>
+                                <q xml:lang="gez">በር<unclear>ና</unclear>ባስ።</q>
+                            </decoNote>
+                            
+                            <decoNote xml:id="d9" type="miniature">
+                                <locus target="#146v"/>
+                                <desc></desc>
+                                <q xml:lang="gez">እግዚእነ፡ ዘከመ፡ ሰዐሞ<unclear>፡</unclear> ለዮሐንስ<supplied reason="subaudible">፡</supplied></q>
+                                <q xml:lang="gez">ሚካኤል፡ መልአክ።</q>
+                                <q xml:lang="gez">ሥዕለ፡ ጴጥሮስ፡ ወጳውሎስ።</q>
+                                <q xml:lang="gez">መራኁት።</q>
+                                <q xml:lang="gez">ዮሐንስ፡ መጥምቅ፡</q>
+                                <q xml:lang="gez">እንድርያስ፡ ሐዋርያ፡</q>
+                                <q xml:lang="gez">ሰአሉ፡ ለነ።</q>
+                                <q xml:lang="gez">ያዕቆብ<unclear>፡</unclear> እኁሁ፡ ለእግዚእነ፡</q>
+                                <q xml:lang="gez">ማርቆስ። ። ።</q>
+                                <q xml:lang="gez">በርተሎሜዎስ፡</q>
+                                <q xml:lang="gez">ማቴዎስ፡</q>
+                                <q xml:lang="gez">ቶማስ፡</q>
+                                <q xml:lang="gez">ፊልጶስ፡</q>
+                            </decoNote>
+                            
+                        </decoDesc>
+                        
+                        <additions>
+                            <list>
+
+                                
+                                <item xml:id="e3">
+                                    There are frequent line takeups and line takedowns (<locus target="#1v #7r #8r"/>), occasional erasures 
+                                    (<locus target="#15v #16v #17v #22r #35v #70v #89r"/>), and interlinear additions (<locus target="#47r #54v"/>). 
```

Comment:
Isn't three the maximum number of instances per phenomenon? I would keep three representative instances and add "e.g."